source/docker-monitor/monitoring_containers_activity.rst

```diff
@@ -0,0 +1,172 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _docker_containers_activity:
+
+Monitoring containers activity
+==============================
+
+It is possible to monitoring the events on Docker containers enabling the wodle ``docker-listener``.
+
+This wodle catch events as starting, stopping or pausing Docker containers.
+
+Wodle requirements
+------------------
+
+Dependencies
+^^^^^^^^^^^^
+
+- Python >= 2.7
+- Pip
+- docker library for Python
+
+Part of the integration has been implemented in `Python <https://www.python.org/>`_, so we will need to have Python installed with a version 2.7 or higher. 
+
+The correct functioning of the integration implemented in Python requires the presence of certain libraries as `docker library <https://pypi.org/project/docker/>`_.
+
+.. note::
+
+        It is necessary Wauzh 3.7 or higher for use this wodle
+
+
+Installing dependencies
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Pip can be used as Python package manager to install the required module. In order to use it, we will start installing this tool.
+
+
+a) CentOS/RHEL/Fedora:
+
+.. note::
+
+        It may be necessary to enable the EPEL repository
+
+.. code-block:: console
+
+    # yum install python-pip
+
+b) Debian/Ubuntu:
+
+.. code-block:: console
+
+    # apt-get update && apt-get install python-pip
+
+c) From sources:
+
+.. code-block:: console
+
+    # curl -O https://bootstrap.pypa.io/get-pip.py
+    # python get-pip.py
+
+For installing ``docker`` library for Python: 
+
+.. code-block:: console
+
+    # pip install docker
+
+Configuring wodle
+-----------------
+
+It is necessary to have the option ``disabled`` with the value ``no`` in the ``ossec.conf`` and restart the Wazuh agent for using this capability.
+
+.. code-block:: xml
+
+    <wodle name="docker-listener">
+        <interval>2h</interval>
+        <attemps>5</attemps>
```

Comment:
Change it to **attempts**.
https://github.com/wazuh/wazuh/blob/238e145b722968933174a5d1ec08ecb24212063b/src/config/wmodules-docker.c#L16